docs/workers.md

```diff
@@ -251,6 +251,8 @@ information.
     # Presence requests
     ^/_matrix/client/(api/v1|r0|v3|unstable)/presence/
 
+    # User directory queries
+    ^/_matrix/client/(r0|v3|unstable)/user_directory/search$
```

Comment:
nit: For consistency with the comments above, let's end the comment with "requests".
```suggestion
    # User directory search requests
    ^/_matrix/client/(r0|v3|unstable)/user_directory/search$
```